README.md

```diff
@@ -171,6 +172,12 @@ Image| Label | Prediction |
 :-------------------------:|:-------------------------:|:-------------------------:
 ![](data/demos/deeplab/resnet_101_voc2012/image_3.jpg)  |  ![](data/demos/deeplab/resnet_101_voc2012/image_3_label.png) |  ![](data/demos/deeplab/resnet_101_voc2012/image_3_prediction.png)
 
+## Running demo on your own images:<br>
```

Comment:
I am revising it slightly here.

## Custom Demo

Just put some images into `custom_dir` and run the following command in the terminal: 
```
$ python test_any_image.py
```
Results will be written into same folder. Make sure that proper model trained and a checkpoint is saved in models_dir. See the script for details.